### ISSUE TITLE
Some models have been broken since r21381 (or r21370)

### ISSUE THREAD

**Issue by Daemant** [multitheftauto/mtasa-blue#2800]
### Describe the bug

Models that were working fine, unexpectedly broke since the last builds.
DFF loads correctly and without errors, but engineReplaceModel says 'Model ID X replace failed' (ID does not matter).
Because logs don't provide any further information, it isn't clear what the problem is and how to fix these files.

Also some files have another problem: they are working fine in separate resource, but fail as a part of big replacer resource, despite of same md5 of data passed to engineLoadDFF in both cases. This bug may be connected to previous. It is not consistent though, so at first I will try to solve it by myself and maybe post another bugreport, if not.

### Steps to reproduce

Start the attached resource to spawn some objects and replace models for them.
It will spawn 4 chip stacks in front of player and try to replace their models to big mapping, but will fail on replacing three of them.

[mapping_objects.zip](https://github.com/multitheftauto/mtasa-blue/files/9880698/mapping_objects.zip)

### Version

Client: v1.5.9-release-21378 (Windows 7)
Server: v1.5.9-release-21258

### Additional context

![gta_sa 2022-10-27 22-33-55-474](https://user-images.githubusercontent.com/25665420/198334717-5a181697-7b47-4151-b730-82ed13667d68.jpg)

### Relevant log output

```shell
[2022-10-27 21:56:16] WARNING: [tools]\mapping_objects\client.lua:22: Bad usage @ 'engineReplaceModel' [Model ID 1931 replace failed]
[2022-10-27 21:56:17] WARNING: [tools]\mapping_objects\client.lua:22: Bad usage @ 'engineReplaceModel' [Model ID 1932 replace failed]
[2022-10-27 21:56:17] WARNING: [tools]\mapping_objects\client.lua:22: Bad usage @ 'engineReplaceModel' [Model ID 1933 replace failed]
```

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not security related.

**Comment by botder** [comment]
The cause is most likely coming from my change in d999e3e97770a11f4df25cd96c8bbe360c1fd4d9

**Comment by botder** [comment]
Fixed in c667e2ad05c97511e7b5cf63d223f762eea41e10
